.github/workflows/kra-basic-test.yml

```diff
@@ -60,6 +60,42 @@ jobs:
 
           docker exec pki pki-server cert-find
 
+      - name: Check CA security domain
+        run: |
+          # security domain should be enabled (i.e. securitydomain.select=new)
+          cat > expected << EOF
+          securitydomain.checkIP=false
+          securitydomain.checkinterval=300000
+          securitydomain.flushinterval=86400000
+          securitydomain.host=pki.example.com
+          securitydomain.httpport=8080
+          securitydomain.httpsadminport=8443
+          securitydomain.name=EXAMPLE
+          securitydomain.select=new
+          securitydomain.source=ldap
+          EOF
+          docker exec pki pki-server ca-config-find | grep ^securitydomain. | tee actual
```

Comment:
The `expected` file contains a sorted list of element but IIUC `ca-config-find` is not forced to return a sorted list of items (or I have not find that in the code). If this is the case I would add a sort in the pipeline.